WooCommerce/Classes/Authentication/Epilogue/StorePickerViewController.swift

```diff
@@ -474,6 +536,7 @@ extension StorePickerViewController: UITableViewDataSource {
 
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         guard let site = state.site(at: indexPath) else {
+            hideActionButton()
```

Comment:
This is very nice. I know, sometimes, it seems overkill to extract a method with just two lines, but now, when reading this code, it is telling us the intent, we don't need to figure it out by gathering pieces of code here and there.